Week1/homework/js2-exercises/1-books.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Book list</title>
+</head>
+
+<body>
+  <h1>My book list</h1>
+
+  <script>
```

Comment:
In practice it is rare to have script inside of a script tag. Try to keep JavaScript external by moving the code to . a separate js file and using src attribute in script element to reference it for example 
```html
<script src='./1-books.js' type='text/javascript'></script>
```
The separate code makes it easier for web browsers to cache which enhances performance (here is an article about javascript performance optimisation https://www.keycdn.com/blog/javascript-performance)